src/main/java/com/files/management/exception/GlobalExceptionHandler.java

```diff
@@ -1,46 +1,58 @@
-                "timestamp", ZonedDateTime.now().toString(),
-                "status", String.valueOf(HttpStatus.METHOD_NOT_ALLOWED.value()),
-                "error", HttpStatus.METHOD_NOT_ALLOWED.getReasonPhrase(),
-                "message", e.getMessage(),
-                "path", request.getRequestURI());
-        return new ResponseEntity<>(body, HttpStatus.METHOD_NOT_ALLOWED);
-    }
 
-    @ExceptionHandler(value = MethodArgumentNotValidException.class)
-    public ResponseEntity<Map<String, String>> handleMethodArgumentNotValidException(
-            MethodArgumentNotValidException e, HttpServletRequest request
-    ){
-        Map<String, String> body = Map.of(
-                "timestamp", ZonedDateTime.now().toString(),
-                "status", String.valueOf(HttpStatus.BAD_REQUEST.value()),
-                "error", HttpStatus.BAD_REQUEST.getReasonPhrase(),
-                "message", e.getBindingResult().getAllErrors().stream().map(err -> {
-                    if (err instanceof FieldError) {
-                        return String.format("%s: %s", ((FieldError) err).getField(), err.getDefaultMessage());
-                    }
-                    return err.toString();
-                }).collect(Collectors.joining(", ")),
-                "path", request.getRequestURI());
-        return new ResponseEntity<>(body, HttpStatus.BAD_REQUEST);
-    }
+  @ExceptionHandler(DuplicateLocationException.class)
+  public ResponseEntity<Map<String, String>> handleDuplicateLocationException(
+      DuplicateLocationException e, HttpServletRequest request) {
+    Map<String, String> body = Map.of(
+        "timestamp", ZonedDateTime.now().toString(),
+        "status", String.valueOf(HttpStatus.METHOD_NOT_ALLOWED.value()),
+        "error", HttpStatus.METHOD_NOT_ALLOWED.getReasonPhrase(),
+        "message", e.getMessage(),
+        "path", request.getRequestURI());
+    return new ResponseEntity<>(body, HttpStatus.METHOD_NOT_ALLOWED);
+  }
+
+  @ExceptionHandler(value = MethodArgumentNotValidException.class)
+  public ResponseEntity<Map<String, String>> handleMethodArgumentNotValidException(
+      MethodArgumentNotValidException e, HttpServletRequest request
+  ) {
+    Map<String, String> body = Map.of(
+        "timestamp", ZonedDateTime.now().toString(),
+        "status", String.valueOf(HttpStatus.BAD_REQUEST.value()),
+        "error", HttpStatus.BAD_REQUEST.getReasonPhrase(),
+        "message", e.getBindingResult().getAllErrors().stream().map(err -> {
+          if (err instanceof FieldError) {
+            return String.format("%s: %s", ((FieldError) err).getField(), err.getDefaultMessage());
+          }
+          return err.toString();
+        }).collect(Collectors.joining(", ")),
+        "path", request.getRequestURI());
+    return new ResponseEntity<>(body, HttpStatus.BAD_REQUEST);
+  }
+
+  @ExceptionHandler(LocationNotFoundException.class)
+  public ResponseEntity<Map<String, String>> handleMaterialNotFoundException(
```

Comment:
Exceptionの名前と揃えてあげた方が良いですね！
```suggestion
  public ResponseEntity<Map<String, String>> handleLocationNotFoundException(
```

Reply:
失礼しました。参考にしたコードの情報が残っていました。
本当に細かいところまでありがとうございます！どうしてこんなところまで気付けるんですか？私はこういうところに気づくのが苦手で時間を消費することがあるので、コツがあったら教えていただけると嬉しいのですが。

Reply:
修正しました。
a96d39571a1f3b448224f907d68ecb3848667a12

Reply:
この職業についてから、平日は毎日のようにコードを書いたり、コードレビューをしてきましたので、ぱっと見で細かいところまで読めるようになっているのかもしれないですね！笑
長年の訓練だと思います笑